### PR TITLE
Fix a grammatical error  on line 75

### LINKED ISSUE #24410: Grammatical error in delete branches topic
### Code of Conduct

- [X] I have read and agree to the GitHub Docs project's [Code of Conduct](https://github.com/github/docs/blob/main/CODE_OF_CONDUCT.md)

### What article on docs.github.com is affected?

https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-and-deleting-branches-within-your-repository

### What part(s) of the article would you like to see updated?

Point number 3 in the section 'Deleting a Branch'

The sentence is grammatically incorrect. Needs rewriting.

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by adityaraute**
Fixed a grammatical error and rearranged the sentence to ensure clarity

### Why:

Closes #24410 

### What's being changed (if available, include any code snippets, screenshots, or gifs):

image

Point number 3 here is rewritten to be grammatically correct

### Check off the following:

- [x] I have reviewed my changes in staging (look for the "Automatically generated comment" and click the links in the "Preview" column to view your latest changes).
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### CODE REVIEW

**Comment by cmwilson21** [comment]
@adityaraute Thank you for opening a PR and linking it to your issue! ✨ 

I'll get this triaged for review! :zap:

File: content/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-and-deleting-branches-within-your-repository.md [hunk 1]
```diff
@@ -68,7 +68,7 @@ If the branch you want to delete is associated with an open pull request, you mu
 
 {% data reusables.repositories.navigate-to-repo %}
 {% data reusables.repositories.navigate-to-branches %}
-1. Next branch that you want to delete, click {% octicon "trash" aria-label="Delete BRANCH" %}.
+1. Click {% octicon "trash" aria-label="Delete BRANCH" %} next to the branch that you want to delete.
```
Comment (mchammer01) [inline content/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-and-deleting-branches-within-your-repository.md:71]: LGTM, following our content model that mentions the location on the page first, and then mentions the action. Also changed the `aria-label` parameter to follow our latest accessibility guidelines (should indicate the icon name, not the action that the icon allows a user to implement.)
```suggestion
1. Next to the branch that you want to delete, click {% octicon "trash" aria-label="The trash icon" %} .
```

**Review by mchammer01 — COMMENTED** [review]
@adityaraute 👋🏻 - thanks for your contribution ✨ 
I have made a suggestion that I'm going to apply.

**Review by mchammer01 — APPROVED** [review]
LGTM :shipit:

**Pull request merged**